includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -107,11 +108,12 @@ function get_sortable_columns() {
 
 			case 'lessons':
 				$columns = array(
-					'title'         => array( 'title', false ),
-					'course'        => array( 'course', false ),
-					'students'      => array( 'students', false ),
-					'completions'   => array( 'completions', false ),
-					'average_grade' => array( 'average_grade', false ),
+					'title'              => array( 'title', false ),
+					'course'             => array( 'course', false ),
+					'students'           => array( 'students', false ),
+					'completions'        => array( 'completions', false ),
+					'days_to_completion' => array( 'days_to_completion', false ),
```

Comment:
I think we should remove this as being a sortable column since it doesn't work. @merkushin found a similar issue with the _Last Activity_ column not sorting properly, so we can address it later if we have time as it's lower priority.